packages/api/cms-api/src/user-management/current-user.ts

```diff
@@ -0,0 +1,93 @@
+import { Field, ObjectType } from "@nestjs/graphql";
+import { GraphQLJSONObject } from "graphql-type-json";
+
+import { ContentScope } from "../common/decorators/content-scope.interface";
+import { PermissionConfiguration } from "./user-management.types";
+
+interface CurrentUserInterface {
```

Comment:
there is already a CurrentUserInterface in `auth/current-user/current-user.ts`, should this interface extend it? Why do we need it at all?

(the existing CurrentUserInterface is ment to be extended in the application with module augmentation)